Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.7
```

Comment:
I don't want to change this just yet (this will fail package resolution if at least the Swift version specified is not used). We will also need to update Catapult before we can do this; I will take on Catapult next week.